c7n/resources/emr.py

```diff
@@ -284,3 +284,73 @@ def query(self):
             value = [self.value]
 
         return {'Name': self.key, 'Values': value}
+
+
+@resources.register('emr-block-public-access-configuration')
+class EMRBlockPublicAccessConfiguration(QueryResourceManager):
+    """Resource manager for EMR block public access configuration
+    """
+
+    class resource_type(TypeInfo):
+        service = 'emr'
+        enum_spec = ('get_block_public_access_configuration', None, None)
+        date = "BlockPublicAccessConfigurationMetadata.CreationDateTime"
+        name = id = 'BlockPublicAccessConfigurationMetadata.CreatedByArn'
+        arn_type = 'emr'
+        permission_prefix = 'elasticmapreduce'
+
+    def augment(self, resource):
+        config = resource
+        config.pop('ResponseMetadata')
```

Comment:
What is the purpose of popping off ResponseMetadata here? is it to minimize what is recorded in the tests?

Reply:
its so that it doesn't get into resources.json